src/util/internals/getRandomInt.test.ts

```diff
@@ -0,0 +1,33 @@
+import { getRandomInt } from './getRandomInt';
+
+const originalMathRandom = global.Math.random;
+
+describe('getRandomInt', () => {
+  beforeAll(() => {
+    global.Math.random = jest.fn(() => 0.1);
+  });
+  afterAll(() => {
+    global.Math.random = originalMathRandom;
+  });
+  it('return a number between min and max', () => {
+    (global.Math.random as jest.Mock).mockReturnValue(0.1);
+    const semiRandom = getRandomInt(0, 10);
+    expect(semiRandom).toBe(1);
+  });
```

Comment:
I don't understand why the beforeAll/afterAll if this test is using mocks
I think we should not do that anymore and use only mocks, stubs, spies whatever

Reply:
i don't think you can mock random differently than this.
Because Math is not an import.

Reply:
So i would like to add sinon
Sinon dies that so easily